web-client/src/modules/landing-page/pages/AboutPage.tsx

```diff
@@ -49,6 +58,79 @@ const AboutPage: React.FC = (): JSX.Element => (
       </MissionContainer>
       <SolvingImage src={problemSolve} /> <br />
     </MiddleSection>
+    <MiddleSection>
+      <TeamContainer>
+        <TeamText>
+          <h1> Meet Our Team </h1>
+          <br />
+          <p>
+            Hailing from LA to Morocco to Portugal to Bangladesh, we’re an
+            international collective of volunteers from all walks of life,
+            united to ensure that everyone, everywhere can Reach4Help.
+          </p>
+        </TeamText>
+      </TeamContainer>
+    </MiddleSection>
+    <TeamImageContainer>
```

Comment:
Could the `CoontributorImage` components be implemented with a `.map`? If yes, it'll require lesser amount of code.

Reply:
Right you are, Waris. I think I [suggested something similar](https://github.com/reach4help/reach4help/pull/1041#issuecomment-733675062) a bit ago but completely forgot about it 😅